.github/workflows/ue-docker.yml

```diff
@@ -0,0 +1,79 @@
+name: Build UE Windows Docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      ue_version:
+        description: Select Unreal Engine version
+        required: true
+        type: choice
+        options:
+        - 4.27
+        - 5.0
+        - 5.1
+        - 5.2
+        - 5.3
+        - 5.4
+        - 5.5
+
+jobs:
+  build:
+    runs-on: windows-latest-16-cores
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Building docker images for UE 5.4 and newer require extra disk space
+      - name: Clean up disk space before Docker build        
+        if: ${{ inputs.ue_version == '5.4' || inputs.ue_version == '5.5' }}
```

Comment:
looks like this is going to be necessary all the time, going forward so maybe worth removing the condition.